src/Orleans.TestingHost/Utils/NullTelemetryProducer.cs

```diff
@@ -9,6 +9,8 @@ namespace Orleans.TestingHost.Utils
     /// </summary>
     public class NullTelemetryProducer : ITelemetryProducer
     {
+        public static NullTelemetryProducer Instance = new NullTelemetryProducer();
```

Comment:
make this a get-only field

Reply:
And by extension, make the constructor private :)

Reply:
yeah I thought of that. But then I looked at example at NullLoggerFactory, it has an Instance property, but it also has a public constructor.  I guess it will be nice to have the option to construct one. 

I don't have strong option either way, just mention my thinking behind this. And if anyone know the benefits of have a public constructor in this case.